### DISCUSSION TITLE
sigs.securityonion.net generates a server error

### DISCUSSION THREAD

**Discussion by Alex01000100** [Security-Onion-Solutions/securityonion#11650] [2.4]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

40GB

### Storage for /

200

### Storage for /nsm

I don't know -> permission denied

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Dear all.,

I am new to Security Onion and I tried (many times) and try to implement 2.4 in VMware Workstation Player 17 (17.5.0 build-22583795) but it always fails.

- Salt Status: Data failed to compile. The function "state.highstate" is running as PID...and started...with jid...
- Additional info: the IP of the web interface doesn't appear in ARP table. 
The 2 Networkadpaters in VM are set to NAT and Host-only as recommended in Youtube Video:
Security Onion Essentials 2.4 - Security Onion Installation, Part 1, I used to have: bridged in both NW-Adapters.
- When it reboots, it takes extremely long, like hours: what I see is kind of a flapping port on sobridge:
1. eth0 gets renamed 
2. various change the states from disable, blocking and forwarding and then
3. enters promiscuous mode
4. again various changes
5. left promiscuous mode 
6. rename again (1)
= this goes for hours and at the end: journal header limits reached.

It was running with 16.5 but then I did this update to 17. I noted that in the implementation process, there is a point where sigs.securityonion.net/ is tried to connect 5 times but can't.
Checking that URL on other devices, it generates a server error. 

Bing AI meant:

The message you’ve received is a standard error message from a server.

In summary, this error message is telling you that your request was denied by the server. If you believe this is an error, you may want to contact the service provider or check your permissions.

Maybe I have just to wait for an update 2.4.1?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by cm-ops** [comment]
When you installed did it complete successfully?

**Reply by Alex01000100** [reply]
Thanks for your feedback.

I have not noted an error. To me, it looks like, it completed the implementation.
Almost the end of the implementation, there is this overview saying: "the following options have been set, do you like to proceed".

**Reply by Alex01000100** [reply]
I installed it again and at the end there was an error. So, the implementation did finish with errors. And here is the error.log:
Error: Failed to download metadata for repo 'securityonionsync': CAnnot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
[ERROR]: Command '/usr/sbin/so-yara-download' failed with return code: 1
[ERROR]: stdout: Server returned 000 status code.
[ERROR]: retcode 1
[ERROR]: {'pid': 37895, 'retcode': 1, 'stdout: 'Server returned 000 status code. \nNo connectivity to Github...exiting...', 'stderr': ''}
Result: False
[ERROR]: Command '/usr/sbin/so-rule-update' failed with return code: 1
[ERROR]: retcode: 1
[ERROR]: {'pid': 74476, 'retcode': 1, 'stdout':  '', 'stderr': ''}
Result: False

**Reply by cm-ops** [reply]
Does this node have Internet connectivity?

**Reply by Alex01000100** [reply]
I checked this and observed that all other internet downloads work properly, except sigs.securityonion.net. I checked AV, FW, IPS and so on but there are not entries that something has been blocked.

**Reply by cm-ops** [reply]
Are you able to resolve `sigs.securityonion.net`?

**Reply by Alex01000100** [reply]
Yes on the host system and on other devices:

Non-authoritative answer:
Name:    d15e1rs01gh6d2.cloudfront.net
Addresses:  2600:9000:25a2:5c00:1f:6bbb:9d80:93a1
          2600:9000:25a2:3e00:1f:6bbb:9d80:93a1
          2600:9000:25a2:e000:1f:6bbb:9d80:93a1
          2600:9000:25a2:7400:1f:6bbb:9d80:93a1
          2600:9000:25a2:d200:1f:6bbb:9d80:93a1
          2600:9000:25a2:a600:1f:6bbb:9d80:93a1
          2600:9000:25a2:1e00:1f:6bbb:9d80:93a1
          2600:9000:25a2:1400:1f:6bbb:9d80:93a1
          18.165.183.36
          18.165.183.75
          18.165.183.79
          18.165.183.103
Aliases:  sigs.securityonion.net
(Result from the Hostsystem: Win11)

I did implement the same in virtualbox, and remove AV with the same result.
I will link the NICs "somewhere else" and try again and then I plan to re-install the whole host.

**Reply by Alex01000100** [reply]
Ok, I tried it all and came to the conclusion, that the recommendation in the following video, doesn't work for me.

"The 2 Networkadpaters in VM are set to NAT and Host-only as recommended in Youtube Video:
Security Onion Essentials 2.4 - Security Onion Installation, Part 1"

I reverted to bridged on both NW-Adapters, in VM and it works now.

**Reply by Alex01000100** [reply]
Now, I can't access web-interface of my Security Onion after a reboot where I add some RAM. However, I have seen, there is already an ongoing discussion, so, I will check there.

**Reply by dougburks** [reply]
NAT should work fine for the management interface as long as you correctly configure the IP address, subnet mask, gateway, and DNS settings.

**Reply by Alex01000100** [reply]
The management interface is now also reachable, I had that port in the wrong Vlan configured. I reconfigured also the port-mirror (span) port. I am now at a level where it works  and I will do an improvement second round in the future :-) Thx Doug.
